packages/react-sdk/src/client.ts

```diff
@@ -0,0 +1,595 @@
+/**
+ * Copyright 2019, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as optimizely from '@optimizely/optimizely-sdk'
+import * as logging from '@optimizely/js-sdk-logging'
+
+const logger = logging.getLogger('ReactSDK')
+
+export type VariableValuesObject = {
+  [key: string]: boolean | number | string | null
+}
+
+export type UserAttributes = { [attribute: string]: any }
```

Comment:
This is already exported by the `optimizely-sdk` package. We refer to it and re-export that definition.